lib/reporter.js

```diff
@@ -109,15 +109,15 @@ class JunitReporter extends events.EventEmitter {
                         .time(suite.duration / 1000)
                         .property('specId', specId)
                         .property('suiteName', suite.title)
-                        .property('capabilities', capabilities.sanitizedCapabilities)
+                        // .property('capabilities', capabilities.sanitizedCapabilities)
```

Comment:
why?

Reply:
This should be an option not a default. Ideally set in the config.

Reply:
ok make options to control it.